src/app/settings/organization-plans.component.ts

```diff
@@ -159,6 +159,16 @@ export class OrganizationPlansComponent implements OnInit {
         return subTotal;
     }
 
+    get taxCharges() {
+        return this.taxComponent != null && this.taxComponent.taxRate != null ?
+            (this.taxComponent.taxRate / 100) * this.subtotal :
+            0;
+    }
+
+    get total() {
+        return this.subtotal + this.taxCharges || 0;
```

Comment:
Does this need logical grouping via parens()?

Reply:
[Looks like math always beats conditionals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence) (TIL), but I feel like parens are still appropriate for readability. Added!